core/tests/revert-test/tests/revert-and-restart.test.ts

```diff
@@ -232,8 +279,8 @@ describe('Block reverting test', function () {
         await killServerAndWaitForShutdown(tester);
 
         // Run again.
-        utils.background(`zk server --components=${components}`, [null, logs, logs]);
-        await utils.sleep(10);
+        runServerInBackground({ components: [components], stdio: [null, logs, logs], cwd: pathToHome });
+        await utils.sleep(30);
```

Comment:
why?

Reply:
Sleep time was increased to make tests more reliable. With only 10 s of sleep time, tests sometime failed.

Reply:
hm, weird.. but it will increase the time of the test by it self